### PR TITLE
[INLONG-4824][Log] Print line numbers in log info for all modules

### LINKED ISSUE #4824: [Improve][Agent] Log output has no line numbers
### Description

Agent module log output has no line numbers. 
image

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Greedyu**
Add line numbers to Agent module log output

- Fixes #4824

### CODE REVIEW

File: inlong-agent/conf/log4j2.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
 <configuration status="WARN" monitorInterval="30">
     <Properties>
         <property name="basePath">${sys:agent.home}/logs</property>
-        <property name="log_pattern">%d{yyyy-MM-dd HH:mm:ss.SSS} -%5p ${PID:-} [%15.15t] %-30.30C{1.} : %m%n</property>
+        <property name="log_pattern">%d{yyyy-MM-dd HH:mm:ss.SSS} -%5p ${PID:-} [%15.15t] %-30.30C{1.}:%L %m%n</property>
```
Comment (healchow) [inline inlong-agent/conf/log4j2.xml:23]: Are there any other log configurations of the agent module that need to be modified?

**Comment by dockerzhang** [comment]
@Greedyu please check the log4j configurations for the `DataProxy` and `Manager` at the same time, thanks.

File: inlong-agent/conf/log4j2.xml (hunk 1 above)
Comment (Greedyu) [reply inlong-agent/conf/log4j2.xml:23]: The agent module is not adjusted first, and the logs that other modules do not print line numbers are changed
wecom-temp-b576ae7825a12152918685d5fa3d7d8b

**Comment by Greedyu** [comment]
> configurations

resolve

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Review by pocozh — APPROVED** [review]
+1

**Pull request merged**